php-checks/src/main/java/org/sonar/php/checks/utils/FunctionUtils.java

```diff
@@ -43,6 +43,7 @@
 
 public class FunctionUtils {
 
+  // fixme (@lena) refactor this class
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1134)